pysimplesoap/transport/transport.py

```diff
@@ -1,273 +1,273 @@
-#!/usr/bin/python
```

Comment:
Because urllib2 and urllib3 are both transports, they can both stay in the transport.py. Python style recommends a flat structure, as opposed to java which splits the classes out. https://www.python.org/dev/peps/pep-0423/#avoid-deep-nesting